code/go/0chain.net/blobbercore/allocation/renamefilechange.go

```diff
@@ -24,7 +24,7 @@ func (rf *RenameFileChange) DeleteTempFile() error {
 	return nil
 }
 
-func (rf *RenameFileChange) ApplyChange(ctx context.Context, rootRef *reference.Ref, change *AllocationChange,
+func (rf *RenameFileChange) applyChange(ctx context.Context, rootRef *reference.Ref, change *AllocationChange,
```

Comment:
Is this change required?

Reply:
We're trying not to re-write the code for conductor tests, that's why we lower-cased the fn. that applies the main logic and kept the upper-case named fn. to be a switch between main flow and conductor testing flow. Is this contradicting with any other code?